src/Cthulhu.jl

```diff
@@ -108,34 +109,15 @@ const descend = descend_code_typed
 # src/reflection.jl has the tools to discover methods
 # src/ui.jl provides the user facing interface to which _descend responds
 ##
-function _descend(mi::MethodInstance; iswarn::Bool, params=current_params(), optimize::Bool=true, kwargs...)
+function _descend(mi::MethodInstance; iswarn::Bool, params=current_params(), optimize::Bool=true, debuginfo=true, kwargs...)
```

Comment:
`debuginfo` is not a boolean, it is a symbol that is either `:source`, `:default`, or `:none` (I think :P). Potentially there are other extensions like `:compact` in the future

Reply:
I am going to need to look at this again.
Sometimes it changes?
Maybe I should pull those changes out.

I would like to see a bunch of the UI code seperated.